src/opi/input/structures/structure.py

```diff
@@ -481,7 +481,11 @@ def from_xyz_buffer(
             # > Line should have at least 4 columns
             atom_cols = line.split()
             if len(atom_cols) < 4:
-                raise ValueError(f"Line {iline}: Invalidly formatted coordinate line")
+                # > skip line if Structure is already complete and line is empty
+                if natoms == len(atoms) and len(atom_cols) == 0:
+                    continue
+                else:
+                    raise ValueError(f"Line {iline}: Invalidly formatted coordinate line")
```

Comment:
Don't just skip the line.
I would suggest to `break` here, otherwise other issues might occur because we just blindly jump to the next line. 

[just a suggestion]
If we wanted to be a bit verbose about potential errors, we could do it as follows:
```suggestion
    # > Hit an empty line
    if not atom_cols:
        # > We read the number of atom specified, we can stop here.
        if natoms == len(atoms):
            break
        else:
            raise ValueError("Coordinate block ends after {len(atoms)} atoms, but {natoms} were expected.")
    elif len(atom_cols) < 4:
        raise ValueError(f"Line {iline}: Invalidly formatted coordinate line")
```

Reply:
If we break, we allow anything after an empty line at the end of the structure to be in our input file. If we continue, only further empty lines are allowed. That is why I decided to go with continue because I want to keep the input file unambiguous.